### DISCUSSION TITLE
Warning! not all attachments have been uploaded

### DISCUSSION THREAD

**Discussion by Gabitzzup** [the-djmaze/snappymail#1322] [Q&A] [answered]
Hi please help. i'm on cyberpanel with snappymail 2.28.1
in admin account i cant set attachment size... for now is 0.000476837158203125 mb
if i put other value is save in application.ini but after refresh the same value is display in General- Attachments size .
in the application.ini is saved the new value but every time a refresh or re login at admin this vallue is display 0.000476837158203125.
PLEASE some one tell from where is taken this value ???
i dont think is something wrong with snappy app but something from may server but i cant fiind the glich.
i use php8 and the settings are PHP: upload_max_filesize = 500; post_max_size = 500M
Thank you!

**Comment by Gabitzzup** [comment]
![snappy](https://github.com/the-djmaze/snappymail/assets/112981576/960b15af-fe28-4ef2-9fbb-149ab824cad3)

**Reply by mschilli87** [reply, marked as answer]
@Gabitzzup: Have you tried `upload_max_filesize = 500M` instead of `upload_max_filesize = 500`?

---
*edit:* According to [the PHP manual](https://www.php.net/upload-max-filesize), `upload_max_filesize` defaults to bytes and according to my calculation, 500 B equal roundabout 0.0004768372 MiB. 😉

**Comment by Gabitzzup** [comment]
Thank you so much
 it was in front of me for hours the answer, i didnt see the missing M.
You are the best.
